frontend/workflows/projectCatalog/src/details/index.tsx

```diff
@@ -161,7 +161,9 @@ const Details: React.FC<ProjectDetailsWorkflowProps> = ({ children, chips }) =>
         {/* Column for project quick links */}
         <Hidden smDown>
           <Grid item direction="column">
-            {projectInfo && <QuickLinksCard linkGroups={projectInfo?.linkGroups ?? []} />}
+            {projectInfo && (projectInfo.linkGroups ?? []).length > 0 && (
```

Comment:
wdyt about something like lodash isempty? might be cleaner
```js
_.isEmpty(projectInfo.?linkGroups) && (...)
```

Reply:
Yeah that works, have to utilize a non-null assertion as well but with the additional check it should be good.